### DISCUSSION TITLE
Branding and Identity

### DISCUSSION THREAD

**Discussion by Gazook89** [naturalcrit/homebrewery#2779] [General]
Correct me if I'm wrong, but I'm not sure there is an open Issue already for the possible new logo's that have been pitched?  This thread can be that place where those ideas are collated.   I do think we have other Discussions and Issues open for things like UI design and those can be kept separate.

For logos, my ideal is an SVG that works at small and large sizes, without losing details when set to very small.  And that it can integrate well with regular text-- not necessarily "normal font", just that any text in the logo can be rendered as text vs part of the svg.  Finally, multicolor is fine but it should ideally work in grayscale and even just straight black contours.

At least some of the below are designed around the presumption that the brand is "Homebrewery" rather than the current "The Homebrewery".  

A. Here is something that happened sort of accidentally:   I did not expect the 'h' in 'homebrewery' to fit sit neatly inside the cauldron.  The font is just *Luminari*, which I think is commonly available on computers.
<img width="620" alt="image" src="https://user-images.githubusercontent.com/58999374/230699857-b7c35651-9ec5-4a8e-ac0e-d5711747e8bd.png">
<img height="24" alt="image" src="https://user-images.githubusercontent.com/58999374/230699857-b7c35651-9ec5-4a8e-ac0e-d5711747e8bd.png">
Shorter Icon:
<img height="214" alt="image" src="https://user-images.githubusercontent.com/58999374/230699883-d9e8661a-199a-4e3f-ac21-7d54854515ba.png"> or <img height="214" alt="image" src="https://user-images.githubusercontent.com/58999374/230700001-7f8e4403-c0c2-4e62-bdd8-38ac4ee5841d.png">
<img height="24" alt="image" src="https://user-images.githubusercontent.com/58999374/230699883-d9e8661a-199a-4e3f-ac21-7d54854515ba.png"> or <img height="24" alt="image" src="https://user-images.githubusercontent.com/58999374/230700001-7f8e4403-c0c2-4e62-bdd8-38ac4ee5841d.png">

B.  This is just a non-textual icon that would go wherever.  In my opinion it loses points when it's very small, and it could be simplified a bit, which led to the first (A) option.
<img width="221" alt="image" src="https://user-images.githubusercontent.com/58999374/230700081-6c209680-2cb6-41c9-9b2c-bd2ccf975e6d.png">
<img height="24" alt="image" src="https://user-images.githubusercontent.com/58999374/230700081-6c209680-2cb6-41c9-9b2c-bd2ccf975e6d.png">

C. I think this one looks *classy*....like the sign of some clothing store at an upscale mall in the suburbs.  One nice thing about it is that the logo fits neatly *inside* the name, which means it's taking less space and placement around other items is likely easier?
<img width="615" alt="image" src="https://user-images.githubusercontent.com/58999374/230700422-8e574d9e-74f0-4ae4-a0f3-fc11b993f9d8.png">

D. Similar feel to (C), but with Scaly Sans rather than Mr Eaves and showing the balance with NaturalCrit:
<img width="693" alt="image" src="https://user-images.githubusercontent.com/58999374/230700763-401527e3-1414-4839-8484-29e8edcb2916.png">

E. Again, similar, different font (Copperplate):
<img width="693" alt="image" src="https://user-images.githubusercontent.com/58999374/230701116-b538c348-5f5c-48f1-b65e-ef361b23d832.png">

**Comment by 5e-Cleric** [comment]
Cauldron with HB inside is too good to let slip, i'll hang onto that, maybe make some desings around that and let you know.

if you see this comment and i haven't reached you, feel free to remind me of it.

about Luminari, quick search in google  tells me for its use as a web font, it has a pay once license, but for the use in a logo, can be used as a personal license, which is still 25 ish dollars

source: https://www.fonts.com/font/canada-type/luminari/licenses

**Reply by 5e-Cleric** [reply]
A few suggestions to add to those fonts, all free licenses:

<img width="875" alt="image" src="https://user-images.githubusercontent.com/91662974/230795661-a60bc742-0b7c-41a8-bffa-a5a70e2de5ee.png">

**Reply by dbolack-ab** [reply]
Please no Black Chancery. That's the Comics Sans of fantasy gaming. :)

**Reply by 5e-Cleric** [reply]
In what sense? its definitely my personal favourite here. Also, please add fonts for this.

**Reply by dbolack-ab** [reply]
_Generally over and poorly used._

**Comment by 5e-Cleric** [comment]
### Updating with such fonts:

<img width="714" alt="image" src="https://user-images.githubusercontent.com/91662974/234995730-b72203d0-20b1-4208-8267-50f415f193d1.png">

**Comment by 5e-Cleric** [comment]
This could be a good place to discuss wether having a banner for the subreddit is a good idea

**Comment by 5e-Cleric** [comment]
I tried a gradient suggested by @G-Ambatte to make a quick mockup, the gradient is: 
`to top left, brown 22.5%, gold 40%, white 60%, gold 67.5%, brown 82.5%`
The background is out navigation bar color: `#333`

<img width="400" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/505c3936-d6e0-4cd1-bb3b-e866b5d2dee9">

**Reply by Gazook89** [reply]
I think it looks pretty, and is a cool effect, but am not a fan as a logo (the coloring). In my opinion, flat colors are better as they can be more universally be applied more easily. I’m going to avoid going into a few reasons because I’m about to start kid bedtime. But I can come back to it if anyone wants more details.

**Reply by calculuschild** [reply]
I think this looks ok, but I agree I think I would prefer a flat color.

**Reply by 5e-Cleric** [reply]
For contrast reasons, we should have either a brighter and a darker alternative, or add a stroke or shadow to the logo on backgrounds of a simmilar brightness.

I would go with some color between `brown` and `gold`.

**Reply by 5e-Cleric** [reply]
How about these

| #d86c05 | #c65207 | #f69801 | 
|-|-|-|
|<img width="100" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/0bc33969-c56c-47e6-a2ba-79e0ec2a678e">|<img width="100" src="https://github.com/naturalcrit/homebrewery/assets/91662974/f716a995-e7a0-4a00-9e76-965c4e8027af">|<img width="100" src="https://github.com/naturalcrit/homebrewery/assets/91662974/e5b98ef7-eba2-4d90-a88b-9d216e41db0b">|

**Reply by 5e-Cleric** [reply]
any specific color you feel would work better?

<img width="760" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/2b67503c-6965-4fdd-8774-1cabfa262535">

**Reply by dbolack-ab** [reply]
On this particular background, I feel only the bright yellow and Off-white work well.

**Reply by Kaiburr-kath-hound** [reply]
I think this design is great, personally my favorite of the suggestions

**Comment by Gazook89** [comment]
A quick sketch of another angle on the same idea (letters on a bubbling cauldron):

<img width="302" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/58999374/762fe3e0-e612-4965-89b2-417b38069b69">

**Reply by 5e-Cleric** [reply]
not gonna lie, that's closer to a shield than a cauldron.

Also looks like the logo for a boring network company

**Reply by Gazook89** [reply]
Yeah, it was sort of a “try something different and see what happens” effort. I wouldn’t take it as-is, but I also think it’s an attempt to consider other styles rather than get tunnel vision for the “first idea”. 

Regarding network company, that’s pretty close to the inspiration— I was looking at a collection of many logos of code frameworks (react, svelte, node, js, etc). Many of those are same-y, but they look good when all collected together.

**Comment by 5e-Cleric** [comment]
Adding a few ideas made with AI:

### With Cauldrons:
<img width="236" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/1e6b585f-1206-4ad2-ba9f-981b07e15bec">
<img width="233" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/153aede4-3f93-4f69-a031-dc50c7999e73">
<img width="233" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/a4f86ae8-0511-47a2-83a2-9646e68a1317">

### With scrolls and quills
<img width="253" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/e9743c77-3ee5-48db-aa50-54db2d4929a5">
<img width="233" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/57f99a2c-f0cb-4f39-af66-311697a9a9a0">
<img width="253" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/3f22f139-e9d6-4fd5-86b7-c983a0af43a9">

This weird one with curly spans:
<img width="233" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/688db589-344b-45bd-b7db-4eb9a1ffdb12">
<img width="233" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/0258ce6a-4fd6-4b29-ae3b-695e34ef19aa">

**Reply by 5e-Cleric** [reply]
These are just ideas, we're not making the logo with AI.

**Reply by dbolack-ab** [reply]
I kinda like the Curly face, though not the accompanying font.

**Reply by 5e-Cleric** [reply]
looks like satan + Steven Seagal, but i do find it funny and kind of fitting

**Reply by dbolack-ab** [reply]
yeah, but it does seem like a thing that could be riffed on. Themed hats or something.

**Reply by Gazook89** [reply]
I kind of like the `{H}` just by itself-- like the last image, but without the circle and triangles.    The circle and triangles are fine, but to me they are "extra", and build on the core of `{H}`...they could be added to certain designs, but the *brand* would be just the brackets and h.  

The `{H}` is theme/genre agnostic, simple, and it recalls the unique syntax of HB.  It's *almost* like the typical "code" icon `</>`. 

It is flexible:   as @dbolack-ab mentioned a fondness for the "curly face" above it, I could see the face being a "mascot" that is a thematic extension of `{H}`.  Perhaps a whole set of emoji could be made with faces inside curly brackets.  That's not a real suggestion, but I could see it happening.

**Reply by 5e-Cleric** [reply]
![image](https://github.com/user-attachments/assets/42133769-df95-463d-a67e-6ee724b42bbe)

Generated in Adobe Illustrator with Firefly, and lots of hand editing.

**Reply by dbolack-ab** [reply]
> ![image](https://private-user-images.githubusercontent.com/91662974/377013231-42133769-df95-463d-a67e-6ee724b42bbe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5NzE2MzUsIm5iZiI6MTczMzk3MTMzNSwicGF0aCI6Ii85MTY2Mjk3NC8zNzcwMTMyMzEtNDIxMzM3NjktZGY5NS00NjNkLWE2N2UtNmVlNzI0YjQyYmJlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEyVDAyNDIxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYzZjk3ODE5NDA1YzI5MzQyNTQ3Njg5NGZjZjJhNGRjMmUwODFkYTk1MjhmZmY3OTY1MzY5N2QzYjNjMDBkYTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tixYpMQko7U-bvOyaOIJ_Nwaix27g2cS9OSXHxU1RCI)
> 
> Generated in Adobe Illustrator with Firefly, and lots of hand editing.

I was reskimming this and I wondered how hard it would be to make a die shaped Cauldron? Maybe a 12 instead of a 20.

![dicauldron](https://github.com/user-attachments/assets/867fc482-82f3-4efb-bba1-3d282f024c76)

**Reply by 5e-Cleric** [reply]
Either this image is malformed or you uploaded it wrong

**Reply by lucastucious** [reply]
> Either this image is malformed or you uploaded it wrong

You have GitHub in dark so you can't see it 😂

**Reply by Kaiburr-kath-hound** [reply]
Ah I see it now (I also was confused by the image, curse you dark mode); you took the d20 die and just put a white block on top to imitate an open-topped cauldron shape.

**Reply by 5e-Cleric** [reply]
It still is a gigantic d20 with a white block, alright

**Reply by dbolack-ab** [reply]
Was a quick-and-dirty to illustrate the point which it seems to have done poorly.

**Comment by Gazook89** [comment]
Here is a quick mockup of another idea I had:

<img width="332" alt="image" src="https://github.com/user-attachments/assets/aeb72c63-2eee-4489-bb59-4d89653f81d6">

It could be refined further but just wanted to throw up something quick.

**Reply by 5e-Cleric** [reply]
i like the concept, not so much the color or the unequal H

**Reply by Gazook89** [reply]
Yeah the color was just already the loaded color in my 'palette' and I didn't want to fuss with finding the 'correct' color.  I spent about 1 minute thinking about font.

**Reply by dbolack-ab** [reply]
> Here is a quick mockup of another idea I had:
> 
> <img alt="image" width="332" src="https://private-user-images.githubusercontent.com/58999374/369046865-aeb72c63-2eee-4489-bb59-4d89653f81d6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgwNTM1MzksIm5iZiI6MTcyODA1MzIzOSwicGF0aCI6Ii81ODk5OTM3NC8zNjkwNDY4NjUtYWViNzJjNjMtMmVlZS00NDg5LWJiNTktNGQ4OTY1M2Y4MWQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA0VDE0NDcxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQwYzU5NzFiYmIyYmFjYTk4NTYyMzc5ZjI2M2E4NGEzNGYyMjM3Yzc5YTljMmViM2RjYTU3NjM5OGI0NDlhNDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.azc5q8q2B2Wbwd8VoZz5_sfukf1-2yHV4Lc7G7_7Lr0">
> It could be refined further but just wanted to throw up something quick.

What does it look like just stroked, drop all of the fills.

**Reply by lucastucious** [reply]
> > Here is a quick mockup of another idea I had:
> > <img alt="image" width="332" src="https://private-user-images.githubusercontent.com/58999374/369046865-aeb72c63-2eee-4489-bb59-4d89653f81d6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgwNTM1MzksIm5iZiI6MTcyODA1MzIzOSwicGF0aCI6Ii81ODk5OTM3NC8zNjkwNDY4NjUtYWViNzJjNjMtMmVlZS00NDg5LWJiNTktNGQ4OTY1M2Y4MWQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA0VDE0NDcxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQwYzU5NzFiYmIyYmFjYTk4NTYyMzc5ZjI2M2E4NGEzNGYyMjM3Yzc5YTljMmViM2RjYTU3NjM5OGI0NDlhNDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.azc5q8q2B2Wbwd8VoZz5_sfukf1-2yHV4Lc7G7_7Lr0">
> > It could be refined further but just wanted to throw up something quick.
> 
> What does it look like just stroked, drop all of the fills.

<img src="https://github.com/user-attachments/assets/d653d9f9-6afd-4b9e-964c-d98827375280" width="100" height="100">

Like this ?
More example/test
<img src="https://github.com/user-attachments/assets/77685ef1-7586-4c8e-9271-238833884aa5" width="100" >
<img src="https://github.com/user-attachments/assets/be4cc781-f0d8-4666-9aff-f6682fdf69d3" width="100" >

**Reply by 5e-Cleric** [reply]
noice

**Reply by dbolack-ab** [reply]
I like this.

**Reply by 5e-Cleric** [reply]
The H inside Cauldron has been deemed worthy of being our new logo! we need a more refined version of it though, what was the font? Is there some variant with another cauldron? that one feelss a bit cartoonish

**Reply by Kaiburr-kath-hound** [reply]
Morris Roman has a simpler design with a very similar feel to the one in the original mock up

**Reply by lucastucious** [reply]
@5e-Cleric The font for the H is Alike https://fonts.google.com/specimen/Alike
<img alt="hblogotest_white" width="332" src="https://github.com/user-attachments/assets/109962df-e932-43a5-8de5-3e0f5cdc19f7">

<img alt="hblogotest_dark" width="332" src="https://github.com/user-attachments/assets/e226edf0-747c-4074-bce7-984d497e7003">

and a test with a less cartoony cauldron :
<img alt="hblogotest_cauldron2_black" width="332" src="https://github.com/user-attachments/assets/02311f08-65ce-4699-8bae-51ee4bed7dce">
without handles
<img alt="hblogotest_cauldron2_black" width="332" src="https://github.com/user-attachments/assets/42b0e49a-0701-4bbe-928a-abd767b907d8">

**Comment by 5e-Cleric** [comment]
Another option, following the markdown pattern (the markdown logo is the letter M and a down-arrow)

In white, and black, two options:

<img src="https://github.com/user-attachments/assets/a2f1220f-e1ce-49e4-bf25-a738a02560f1" width="250" height="250"/>

The latter would definitely need some refining.

**Reply by 5e-Cleric** [reply]
To clarify, Homebrewery = Home + Beer icons

**Reply by 5e-Cleric** [reply]
another option would be a home icon and a d20 icon inside it.

**Comment by 5e-Cleric** [comment]
Another idea, combining {} + H :

![image](https://github.com/user-attachments/assets/f4c79b32-266d-4b23-8518-17002deb4eb2)

**Reply by Gazook89** [reply]
Now I’m thinking about a beer stein, with an H on it, with a single curly bracket for the handle.

**Reply by 5e-Cleric** [reply]
![image](https://github.com/user-attachments/assets/d45b47dd-bdab-42a5-ae38-c6ce5ead7e43)

**Reply by dbolack-ab** [reply]
I like this if this is the kind of icon that is desired. It is clean.

I do worry, sometimes, that the "Beerier" ideas come of as not immediately reading as *not* for the Beer use of the term.

**Reply by 5e-Cleric** [reply]
I do worry about using beer for the icons as well, but the brand is called the Homebrewery, hence why i suggested the house + beer one

**Reply by dbolack-ab** [reply]
That is an argument to consider a new name. A weak one, but...

**Reply by dbolack-ab** [reply]
I did say it was a weak one. Homebrewery is simultaneously a great and terrible name.

**Comment by Gazook89** [comment]
Different idea, not really sure what i think of it but, as always, just feeding the idea machine:

## Fountain Pen and Bracket

<img width="195" alt="image" src="https://github.com/user-attachments/assets/c603335c-27c6-49b6-b07b-89e63100375a">

<img width="195" alt="image" src="https://github.com/user-attachments/assets/23e10c4a-459b-4d69-b2b4-c90325297b89">

<img width="195" alt="image" src="https://github.com/user-attachments/assets/e81585ff-63c5-4fed-9c5e-23d1fe36ded3">

<img width="195" alt="image" src="https://github.com/user-attachments/assets/b3bca835-b42c-48a6-a032-ea02147213e7">


----

<img width="32" alt="image" src="https://github.com/user-attachments/assets/c603335c-27c6-49b6-b07b-89e63100375a">

<img width="32" alt="image" src="https://github.com/user-attachments/assets/23e10c4a-459b-4d69-b2b4-c90325297b89">

<img width="32" alt="image" src="https://github.com/user-attachments/assets/e81585ff-63c5-4fed-9c5e-23d1fe36ded3">


<img width="32" alt="image" src="https://github.com/user-attachments/assets/b3bca835-b42c-48a6-a032-ea02147213e7">

---

(I can see that the fountain pen looks a little "knife-y" but this is a quick 5 minute sketch).

Possibly this can be used as a different icon, maybe for the "Brew Editor" tab if we ever add icons to that.

**Reply by 5e-Cleric** [reply]
as seen in the smaller versions, you can't much see what it is, it's an okay logo but a bad icon

**Comment by Gazook89** [comment]
## Curly Bracket as Open Book

<img width="192" alt="image" src="https://github.com/user-attachments/assets/4a50b85b-7f70-4210-bc79-d506135e576f">

<img width="32" alt="image" src="https://github.com/user-attachments/assets/4a50b85b-7f70-4210-bc79-d506135e576f">

**Comment by Gazook89** [comment]
## Letterpress Moveable Type Blocks

<img width="195" alt="image" src="https://github.com/user-attachments/assets/f16e431a-6bec-4b3c-bdcf-ef1c231f99fb">

<img width="32" alt="image" src="https://github.com/user-attachments/assets/f16e431a-6bec-4b3c-bdcf-ef1c231f99fb">

**Comment by lucastucious** [comment]
I kinda love the cauldron logo, why changing it ? Imo it's the most recognizable one.

Here is more idea based on what I imagine for the logo 
<img width=300 src='https://github.com/user-attachments/assets/dd48788f-223e-48a5-b50c-53ed4df01fcf' /><img width=300 src='https://github.com/user-attachments/assets/559be37b-c491-420e-9e95-8353ba3a1a49' />

**Reply by Gazook89** [reply]
I'm not sure it'll ever change.  I think the original motivation here was that the cauldron logo is used in *one spot*, which is on the https://www.naturalcrit.com page which is seldomly visited (the direct link to the homebrewery home page is most often shared).  Instead, it is the 'naturalcrit die' that is most commonly used-- on the subreddit banner, in the brew image placeholder, in the menubar, etc.   So I was thinking about flipping that dynamic, and in the process revisiting what the logos could be.

Also, just fyi, i edited your comment just to reduce the displayed size of the images a bit.

**Reply by 5e-Cleric** [reply]
it looks alright, but we need the logos to work as icons, so that is too much clutter, you'd need to indentify it at a size of 150px max

**Comment by Kaiburr-kath-hound** [comment]
Late to the party, I know, but I really like the curly bracket integration into the logo. Has anyone toyed with making the “B” in an HB abbreviated logo as the curly bracket? Something like “Hl}”, but with a more stylized and curvy bracket?

**Reply by 5e-Cleric** [reply]
## Graphic design is my PaSsIoN

![image](https://github.com/user-attachments/assets/4484d856-4524-4166-955a-99768128be0b)

**Reply by Gazook89** [reply]
My quick attempt before i have to go make lunch: 

<img width="224" alt="image" src="https://github.com/user-attachments/assets/0375c9a9-2569-4de6-8ec0-685cb00442f8">

<img width="224" alt="image" src="https://github.com/user-attachments/assets/c1a148be-3769-4e56-b213-5fd36930d247">


I don't love it, particularly the `H`, but that is also where i spent the least effort.

**Reply by 5e-Cleric** [reply]
Is that a crossbow?

**Reply by Kaiburr-kath-hound** [reply]
I like the crossbow-esque idea too, but I love the cauldron too much not to try and incorporate it

**Comment by Gazook89** [comment]
@lucastucious Do you happen to have an SVG of this, and a way to share it:

![image](https://github.com/user-attachments/assets/7bfb817b-afa9-4042-9d9d-3c1be0528810)

**Reply by lucastucious** [reply]
Yes of course. I will send it as soon as possible

**Comment by Gazook89** [comment]
I just realized that the "Code" font, what we have labelled as "Code Light" and "Code Bold", does not have an open license, and we use it for NaturalCrit in the upper left of HB app.  And as the headers of the Vault.

**Reply by 5e-Cleric** [reply]
lmao
